pipeline.yaml

```diff
@@ -467,7 +467,7 @@ deploymentSpec:
         - -c
         - "\nif ! [ -x \"$(command -v pip)\" ]; then\n    python3 -m ensurepip ||\
           \ python3 -m ensurepip --user || apt-get install python3-pip\nfi\n\nPIP_DISABLE_PIP_VERSION_CHECK=1\
-          \ python3 -m pip install --quiet --no-warn-script-location 'kfp==2.9.0'\
+          \ python3 -m pip install --quiet --no-warn-script-location 'kfp==2.8.0'\
```

Comment:
Does this have anything to do with allowing negative number? If not, could you keep in a separate OR or at least a separate commit? Thanks!

Reply:
3fdfb3aacbbcb0da231c6029f943f7488ab1c476